src/evm/protocol/uniswap_v2/state.rs

```diff
@@ -12,7 +8,9 @@ use crate::{
     },
     safe_math::{safe_add_u256, safe_div_u256, safe_mul_u256, safe_sub_u256},
 };
-use tycho_core::dto::ProtocolStateDelta;
+use ethers::types::U256;
+use std::{any::Any, collections::HashMap};
+use tycho_core::{dto::ProtocolStateDelta, Bytes};
```

Comment:
@louise-poole is going to be mad about this import order 👀 
I thought we were doing:
- std imports
- external imports
- tycho
- local project

Reply:
(I didn't see her review before posting mine 😂 )

Reply:
VM is supported I just haven't added it to the example. Any protocol state that implement the `TryFromWithBlock` trait is automatically supported no need to change anything on the code.